BrainPortal/app/models/cluster_task.rb

```diff
@@ -2049,12 +2049,13 @@ def docker_commands(command_script)
   # Then name of the image will be set in a bash variable 'docker_image_name'
   def load_docker_image_cmd #:nodoc:
     dockerhub_image_name = self.tool_config.containerhub_image_name
-
+    docker_container_index = self.tool_config.container_index_location
```

Comment:
This will crash if `container_index_location` is `nil` when the value is substituted in the `pull` command below.

Reply:
Addressed in https://github.com/aces/cbrain/pull/649/commits/9668f65b4e7cad30cdd00d548f108edaf0f76ffb